app/src/main/java/com/example/numberbaseball/base/BaseActivity.kt

```diff
@@ -17,11 +16,11 @@ abstract class BaseActivity(
         setContentView(layoutId)
     }
 
-    protected fun @receiver:StringRes Int.toast(context: Context){
-        getString(this).toast(context)
+    protected fun @receiver:StringRes Int.toast() {
```

Comment:
Toast는 Activity에서 사용되는 function 이므로, String을 Extension 하지 말아주세요.

여기에서는 다음과 같이 사용해주세요.
```suggestion
    protected fun toast(
        @StringRes
        stringRes: Int 
    ) = toast(this, getString(stringRes))
```

Reply:
49b24e9

수정 완료했습니다.